api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -247,7 +248,7 @@ private DbUser initializeUserIfNeeded() {
             + "@"
             + workbenchConfigProvider.get().googleDirectoryService.gSuiteDomain;
 
-    com.google.api.services.directory.model.User googleUser =
+    User googleUser =
```

Comment:
I noticed that this `User` is unambiguous wherever we use it now (long ago, this was not true) so I made this change as well.  It's an a separate commit, so easy to remove if you don't like it.